testing/mysql/APKBUILD

```
@@ -0,0 +1,299 @@
+# Maintainer:
+pkgname=mysql
+pkgver=8.0.16
+pkgrel=0
+pkgdesc="A very fast and reliable SQL database server"
+
+url="https://www.mysql.com/"
+pkgusers="mysql"
+pkggroups="mysql"
+arch="all"
+license="GPL-2.0-only"
+depends="$pkgname-common"
+depends_dev="openssl-dev zlib-dev"
+makedepends="bison cmake icu-dev libaio-dev libc-dev libedit-dev libevent-dev
+		 lz4-dev libtirpc-dev linux-headers linux-pam-dev musl-dev ncurses-dev
+		 protobuf-dev openssl-dev re2-dev rpcgen rapidjson-dev valgrind-dev
+		 zlib-dev gzip lz4 perl perl-json perl-libwww"
```

```suggestion
		 gzip lz4 perl perl-json perl-libwww boost-dev"
```

Use packaged `boost` instead of a bundled version if possible. This means changing `configure` arguments also.